src/hydra/hydraClient.js

```diff
@@ -98,6 +93,25 @@ export default ({entrypoint, resources = []}, httpClient = fetchHydra) => {
     });
   };
 
+  /**
+   * @param {Object} resource 
+   * @param {Object} data 
+   * 
+   * @returns {Promise}
+   */
+  const transformAORDataToRequestBody = (resource, data = {}) => {
+    resource = resources.find(({name}) => resource === name);
+    if (undefined === resource) {
+      return Promise.resolve(data);
+    }
+
+    return convertAORDataToHydraData(resource, data).then(data => {
+      return undefined === resource.serializeData
```

Comment:
Could you rename `serializeData` to `encodeData`? The logic is pretty same than [Serializer component](https://symfony.com/doc/current/components/serializer.html).

Reply:
:+1: That's very good name!